src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx

```diff
@@ -0,0 +1,204 @@
+    } else if (!values.comments.length < 20) {
+      errors.comments = '❌ Must be at least 20 characters';
+    }
+    if (!values.feature) {
+      errors.feature = '❌ Please select a feature!';
+    }
+    if (values.feedbackType === 'selection' || !values.feedbackType) {
+      errors.feedbackType = '❌ Please select feedback type!';
+    }
+    return errors;
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles['title-wrapper']}>
+        <h2 className={styles.title}>Send Feedback!</h2>
+        <h4 className={styles.subtitle}>
+          Please use this form to share your feedback. If you’re experiencing technical issues,
+          please include details.
+        </h4>
+      </div>
+      <form className={styles.form} onSubmit={handleSubmit}>
+        <div className={styles.inputContainer}>
+          <label htmlFor="username">Name</label>
+          <ErrorMessage message={formErrors.username} />
+          <input
+            id="username"
+            type="text"
+            onChange={handleChange}
+            name="username"
+            value={formData.username}
+          />
+        </div>
+
+        <div className={styles.inputContainer}>
+          <label htmlFor="email">Email</label>
+          <ErrorMessage message={formErrors.email} />
+          <input
+            id="email"
+            type="text"
+            onChange={handleChange}
+            name="email"
+            value={formData.email}
+          />
+        </div>
+
+        <div className={styles.inputContainer}>
+          <label htmlFor="comments">Comments</label>
+          <ErrorMessage message={formErrors.comments} />
+          <textarea
+            id="comments"
+            value={formData.comments}
+            onChange={handleChange}
+            name="comments"
+            rows={5}
+          />
+        </div>
+
+        <div className={styles.checkboxContainer}>
+          <input
+            type="checkbox"
+            id="newsletter"
+            checked={formData.newsletter}
+            onChange={handleChange}
+            name="newsletter"
+          />
+          <label htmlFor="newsletter">Do you wish to join our newsletter?</label>
+        </div>
+
+        <fieldset>
+          <legend>What feature is your feedback about?</legend>
+          <input
+            type="radio"
+            id="currentConditions"
+            name="feature"
+            value="Current Conditions"
+            checked={formData.feature === 'Current Conditions'}
+            onChange={handleChange}
+          />
+          <label htmlFor="currentConditions">Current Conditions</label>
+          <br />
+
+          <input
+            type="radio"
+            id="forecast"
+            name="feature"
+            value="Forecast"
+            checked={formData.feature === 'Forecast'}
+            onChange={handleChange}
+          />
+          <label htmlFor="forecast">Forecast</label>
+          <br />
```

Comment:
br tag is usually used for text formatting (inside `<p></p>` or `<span></span>` tags) not blocks, so would be better to use css instead in this case